packages/button/README.md

```diff
@@ -5,17 +5,19 @@ Mineral UI `Button` component. Find out more about [Mineral UI](https://github.c
 
 ## Installation
 
-1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
-
-2. Install the package
+1. Install the package
 
   ```sh
   npm install --save @mineral-ui/button
   ```
 
+2. Install any missing peer dependencies reported by `npm` or `yarn`.
```

Comment:
Should we follow this same pattern in the main readme?  Subsequently, is the peer-dependencies doc needed anymore?